packages/cli/src/commands/model-instance/content.ts

```diff
@@ -0,0 +1,43 @@
+import { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
+import { Command, type QueryCommandFlags, STREAM_ID_ARG, SYNC_OPTION_FLAG } from '../../command.js'
+import { Flags } from '@oclif/core'
+import fs from 'fs-extra'
+
+type ModelContentFlags = QueryCommandFlags & {
+  output?: string
+}
+
+export default class ModelInstanceContent extends Command<ModelContentFlags, { streamId: string }> {
+  static description =
+    'load a model stream instance with a given stream id and display its contents'
+
+  static args = [STREAM_ID_ARG]
+
+  static flags = {
+    ...Command.flags,
+    output: Flags.string({
+      char: 'o',
+      description: 'path to the file where the model instance content should be saved',
+    }),
+
+    sync: SYNC_OPTION_FLAG,
+  }
+
+  async run(): Promise<void> {
+    try {
+      const mid = await ModelInstanceDocument.load(this.ceramic, this.args.streamId)
+      const midContentAsJSON = JSON.stringify(mid.content, null, 2)
+      if (this.flags.output !== undefined) {
+        const output = this.flags.output
+        await fs.writeFile(output, midContentAsJSON)
```

Comment:
Could you use the `write` function from the [local `fs` module](https://github.com/ceramicstudio/js-glaze/blob/main/packages/cli/src/fs.ts#L12) instead please? It has additional logic to resolve paths and create directories as needed.